_config.yml

```diff
@@ -459,11 +459,13 @@ math:
   mathjax:
     # Use 2.7.1 as default, jsdelivr as default CDN, works everywhere even in China
     cdn: //cdn.jsdelivr.net/npm/mathjax@2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML
-    # For newMathJax CDN (cdnjs.cloudflare.com) with fallback to oldMathJax (cdn.mathjax.org).
-    #cdn: //cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML
```

Comment:
Check this link – it's worked for now, i think no need to remove it.

Reply:
`http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML` just make a redirect to `https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js`, and you can see `console.warn('WARNING: cdn.mathjax.org has been retired. Check https://www.mathjax.org/cdn-shutting-down/ for migration tips.')` in the script.
I suggest to removing it.

Reply:
Ok, remove it. Under your responsibility. =)

Reply:
=)